src/themes/UseOfFunds/FundsContent.scss

```
@@ -132,33 +132,13 @@ $padding-mobile: 16px;
 			position: relative;
 			list-style: none;
 		}
-
-		&-item-hand::before {
-			background: url( 'https://upload.wikimedia.org/wikipedia/commons/f/f2/Icon_hand_holding_heart.svg' );
-		}
-
-		&-item-smartphone::before {
-			background: url( 'https://upload.wikimedia.org/wikipedia/commons/5/53/Icon_letter_W_on_smartphone.svg' );
-		}
-
-		&-item-world::before {
-			background: url( 'https://upload.wikimedia.org/wikipedia/commons/d/dd/Icon_world.svg' );
-		}
-
-		&-item-megaphone::before {
-			background: url( 'https://upload.wikimedia.org/wikipedia/commons/9/9d/Icon_werbung.svg' );
-		}
-
-		&-item-twentyfourseven::before {
-			background: url( 'https://upload.wikimedia.org/wikipedia/commons/8/80/Icon_24_7.svg' );
-		}
 	}
 
 	&-orgchart-text {
 		flex: 1 1 auto;
 	}
-	&-orgchart-image img {
-		padding: 0 50px;
+	&-orgchart-image {
+		text-align: center;
```

We need to keep the padding here:

```suggestion
		text-align: center;
		padding: 0 50px;
```